src/app/components/media/MediaComponentRightPanel.js

```diff
@@ -83,12 +83,14 @@ const MediaComponentRightPanel = ({
         />
         <Tab
           label={
-            <FormattedMessage
-              id="mediaComponent.notes"
-              defaultMessage="{notes_count, plural, =0 {Notes} other {Notes [#]}}"
-              description="Label for the Notes tab, as in text notes"
-              values={{ notes_count: projectMedia.notes_count }}
-            />
+            <span>
```

Comment:
@amoedoamorim here is a different approach, is this what you would suggest going with?

Reply:
Yeah, this is better, thanks!